### ISSUE TITLE
Python 3.6 not building - "The executable /usr/bin/python3.6 does not exist"

### ISSUE THREAD

**Issue by hunt3ri** [Shippable/support#3873]
Hi

I'm trying to use Python 3.6 environment but keeping getting the error:

virtualenv -p /usr/bin/python3.6 /root/venv/3.6 4s  
The executable /usr/bin/python3.6 (from --python=/usr/bin/python3.6) does not exist

Build file is here

https://app.shippable.com/github/thinkWhere/themapcloud-portal/runs/2/1/console

**Comment by a-murphy** [comment]
Some of the oldest Python images don't have Python 3.6 installed.  The easiest solution is to update to a more recent image.  Each subscription has a [machine image](http://docs.shippable.com/platform/tutorial/runtime/ami-overview/) selected that determines the default images used for all projects in that subscription.  You can update this machine image to a more recent one on the [nodes page](http://docs.shippable.com/platform/visibility/subscription/nodes/).  5.3.2 and later should all have Python 3.6, and all use Ubuntu 16.04 images instead of Ubuntu 14.04 by default.  

If you don't want to update your other projects, you can also specify a newer image for just this project in your `shippable.yml` using the `pre_ci_boot` section.  Documentation explaining how to specify an image is available [here](http://docs.shippable.com/ci/custom-docker-image/#pulling-a-public-image).  For example, this will use the most recently released Ubuntu 14.04 Python image:
```YML
build:
  pre_ci_boot:
    image_name: drydock/u14pytall
    image_tag: v5.8.2
```

The disadvantage of choosing a more recent image than the machine image is that you'll have to wait for the image to be pulled from Docker Hub, but in some cases that is preferred over updating the images used by all the projects.

**Comment by hunt3ri** [comment]
Awesome thanks for the quick response.  Have upgraded to the latest node and it's all working, in fact this shaves 4 minutes off our build time :)  Previously I was installing node and python 3.6 now I don't have to

You should let all customers on old nodes know about this :)

**Comment by a-murphy** [comment]
The Python versions available in the v5.8.2 Python image are documented [here](http://docs.shippable.com/platform/runtime/machine-image/ami-v582/#pyt-582).  It includes both 2.7.12 and 3.6.2.
